docs/jobs.rst

```diff
@@ -450,3 +458,26 @@ Job Run States
     The run is in an unknown state. This state could indicate a bug in Tron, or
     an exceptional situation with the infrastructure that requires manual inspection.
     Actions for this job may in fact still be running, but Tron cannot reach them.
+
+
+Troubleshooting
+^^^^^^^^^^^^^^^
+**My job doesn't start even though the trigger are emitted?**
+    Check that both jobs are in the same tron master. A "tron master" refers to a
+    cluster; like tron-norcal-devc, tron-nova-prod, etc. Triggers don't work across
+    tron masters! You can emit the event manually using command line or API
+
+**S3 consistency issues**
+    If your downstream job relies on s3 list to process data you may see it triggered
+    before S3 had finished replicating. This was previously masked by log_done
+    continuously polling S3 to determine if upstream finished. See STREAMINT-269 for
+    details.
+
+.. _shortdate:
+**What does shortdate in triggers mean?**
+    There are two concepts of shortdate here.
+
+    shortdate in triggered_by: this shortdate is technically the run_date,
+    indicating when the tron job runs
+    shortdate in command: this shortdate is used by batch jobs to specify which s3
+    dir it is writing to or polling.
```

Comment:
I think these are better placed in y/tron-faq than here. I think they will be more discoverable and easier for others to edit. They are also very yelpy

Reply:
They already exists on yelp wiki the cross-job dependency page. I move them here mostly because I don't think everyone knows all the different y/links or wiki pages. At least I expect everything to be on Tron doc.